src/System.IO.FileSystem/tests/File/GetSetTimes.cs

```diff
@@ -92,5 +92,20 @@ public void PageFileHasTimes()
                 });
             }
         }
+
+        [Fact]
+        public void SetLastWriteTimeMilliSec()
+        {
+            string firstFile = GetTestFilePath();
+            string secondFile = GetTestFilePath();
+
+            File.WriteAllText(firstFile, "");
+            File.WriteAllText(secondFile, "");
+
+            File.SetLastAccessTimeUtc(secondFile, DateTime.UtcNow);
+            long firstFileTicks = File.GetLastWriteTimeUtc(firstFile).Ticks;
+            long secondFileTicks = File.GetLastWriteTimeUtc(secondFile).Ticks;
```

Comment:
@anipik do you need a test that verifies that after SetLastWriteTime, the milliseconds are non zero (999/1000 times anyway). Presumably existing tests aren't enough since they were passing without this fix.

Reply:
The current tests doesnt check the millisecond attribute for any operation, i think the problem lies here https://github.com/dotnet/corefx/blob/master/src/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs#L51
We are just setting seconds(0) and milliseconds will automatically take value of zero. 
I suggest we just change this to some non-zero value for both seconds and milliseconds.
We can even check ticks in all cases as that will check the accuracy of operations upto 100 NanoSeconds.